docs/setting-up/server/docker.md

```diff
@@ -69,9 +118,50 @@ How to run PMM Server with Docker based on our [Docker image].
         ```sh
         docker exec -t pmm-server bash -c 'grafana-cli --homepath /usr/share/grafana --configOverrides cfg:default.paths.data=/srv/grafana admin reset-admin-password newpass'
         ```
+        
+5. Visit `https://localhost:443` to see the PMM user interface in a web browser. (If you are accessing the docker host remotely, replace `localhost` with the IP or server name of the host.)
+
+### Run Docker with the host directory
+
+1. Pull the image.
+
+    ```sh
+    docker pull percona/pmm-server:2
+    ```
+
+2. Run the image.
+
+    ```sh
+    export DATA_DIR=$HOME/srv
+    docker run -v $DATA_DIR/srv:/srv -d --restart always --publish 80:80 --publish 443:443 --name pmm-server perconalab/pmm-server-fb:PR-2534-517015b
+    ```
+`DATA_DIR` is a directory where you want to store the state for PMM.
+
+3. Change the password for the default `admin` user.
+
+    * For PMM versions 2.27.0 and later:
+
+    ```sh
+    docker exec -t pmm-server change-admin-password <new_password>
+    ```
+
+    * For PMM versions prior to 2.27.0:
+
+        ```sh
+        docker exec -t pmm-server bash -c 'grafana-cli --homepath /usr/share/grafana --configOverrides cfg:default.paths.data=/srv/grafana admin reset-admin-password newpass'
+        ```
```

Comment:
The same comment as for the block above.

Reply:
This is required.